barretenberg/ts/package-lock.json

```diff
@@ -0,0 +1,6222 @@
+{
+  "name": "@aztec/bb.js",
```

Comment:
This file should not be modified I think?

Reply:
Done, I removed it.